docs/product-highlights-and-benefits/live-context.mdx

```diff
@@ -1,65 +1,87 @@
 ---
-title: Workstream Context
-description: Powered by Workstream Pattern Engine, Workstream Context can now understand recent workstream contexts, eliminating the need for manual grounding.
+title: Live Context
+description: Powered by Workstream Pattern Engine, Live Context is the world’s first Temporally Grounded Copilot that understands your workflow.
```

Comment:
"Powered by our Workstream Pattern Engine, Live Context enables the world’s first Temporally Grounded Copilot that understands your unique workflow"

Reply:
updated